INSTALL

```diff
@@ -179,3 +179,6 @@ CONFIG+=no-qssldiffiehellmanparameters
 CONFIG+=no-xboxinput (Mumble, Win32)
  Don't build in support for global shortcuts
  from Xbox controllers via the XInput DLL.
+
+CONFIG+=no-manual-plugin (Mumble)
+ Don't include the built-in manual plugin.
```

Comment:
Should mention positional audio. Otherwise it could be quite confusing for someone not in-the-know about the strange lingo about what is a "plugin" for us. Maybe even another half sentence on what it does.